src/layers/fp64/arc-layer/arc-layer-vertex.glsl

```diff
@@ -20,6 +20,8 @@
 
 #define SHADER_NAME arc-layer-vertex-shader
 
+uniform float ONE; // nvidia workaround
```

Comment:
Consider integrating this the new shader assembly system and just do this in JS when loading the shader
```
  vs: assembleShader(gl, {
    fp64: true,
    vs: glslify()
  })
```